charts/spire/charts/spiffe-csi-driver/values.yaml

```diff
@@ -60,6 +60,13 @@ nameOverride: ""
 ## @param namespaceOverride Namespace to install spiffe-csi-driver
 namespaceOverride: ""
 
+## @param serverNamespaceOverride Override the namespace that the spire-server is installed into
+serverNamespaceOverride: ""
+
+validatingAdmissionPolicy:
+  ## @param validatingAdmissionPolicy.enabled When auto, an upstream driver, and k8s >= 1.30.0 then enabled = true, otherwise false
```

Comment:
Im not understanding the upstream driver part of this comment. Seems like some qualifier is missing.

Reply:
{{- $upstream := eq .Values.pluginName "upstream.csi.spiffe.io" }}

Could word that explicitly. something like this maybe?:
```
  ## @param validatingAdmissionPolicy.enabled When auto, pluginName == "upstream.csi.spiffe.io", and k8s >= 1.30.0 then enabled = true, otherwise false
```

Reply:
Yes thats better

Reply:
Sounds good. Merged.